src/ray/protobuf/node_manager.proto

```diff
@@ -131,6 +131,8 @@ message ReturnWorkerRequest {
   bool disconnect_worker = 3;
   // Whether the worker is exiting and cannot be reused.
   bool worker_exiting = 4;
+  // The error message for disconnect_worker.
```

Comment:
This file change is to add more detail when a worker is returned with was_error = true. I found it currently doesn't give any detail